### ISSUE TITLE
new rule: `svelte/no-add-event-listner`

### ISSUE THREAD

**Issue by baseballyama** [sveltejs/eslint-plugin-svelte#1194]
### Motivation

We need to use `on` instead.
https://svelte.dev/docs/svelte/svelte-events#on

### Description

Since Svelte 5 uses event delegation, using `addEventListener` may cause events to fire in unexpected order.
Therefore, we should always use `on` instead.

### Examples

```svelte
<script>
  import { on } from 'svelte/events';

  onMount(() => {
    // ✓ GOOD
    on(parent, 'click',() => {});
    // ✗ BAD
    parent.addEventListener('click',() => {});
  })
</script>

<div bind:this={parent}>
  <button />
</div>
```
